examples/mysql/getting_started_step_1/src/bin/show_posts.rs

```diff
@@ -0,0 +1,23 @@
+extern crate diesel_demo_step_1_mysql;
+extern crate diesel;
+
+use diesel_demo_step_1_mysql::*;
+use diesel_demo_step_1_mysql::models::*;
```

Comment:
Let's replace `diesel_demo_step_x_mysql` with `self` on all of the lines other than `use diesel_demo_step_x_mysql::*`